finite-state-machine/src/main/java/com/iluwatar/finite/state/machine/Main.java

```diff
@@ -0,0 +1,33 @@
+package com.iluwatar.finite.state.machine;
+
+/**
+ * Main.
+ */
+public final class Main {
```

Comment:
We have the convention to call the main class `App`. I recommend to follow that approach here as well.